source/grid/grid_out.cc

```diff
@@ -2951,6 +2951,46 @@ namespace
     return v;
   }
 
+  /**
+   * Return all lines of non-internal faces in three dimension.
+   */
+  std::vector<typename Triangulation<3, 3>::active_line_iterator>
+  all_co_faces(const Triangulation<3, 3> &tria)
+  {
+    std::vector<typename Triangulation<3, 3>::active_line_iterator> res;
+
+    std::vector<bool> flags;
+    tria.save_user_flags_line(flags);
+    const_cast<Triangulation<3, 3> &>(tria).clear_user_flags_line();
+
+    for (auto face = tria.begin_active_face(); face != tria.end_face(); ++face)
```

Comment:
I think you can now say this:
```suggestion
    for (auto face = tria.active_face_iterators())
```